src/Command/PHPDocCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Moodle Plugin CI package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Copyright (c) 2018 Blackboard Inc. (http://www.blackboard.com)
+ * License http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace MoodlePluginCI\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\ProcessBuilder;
+use MoodlePluginCI\Bridge\MoodlePlugin;
+
+class PHPDocCommand extends AbstractMoodleCommand
+{
+    use ExecuteTrait;
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('phpdoc')
+            ->setDescription('Run Moodle PHPDoc Checker on a plugin');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+        $this->initializeExecute($output, $this->getHelper('process'));
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->outputHeading($output, 'Moodle PHPDoc Checker on %s');
+
+        // We need local_moodlecheck plugin to run this check.
+        $pluginlocation  = __DIR__.'/../../vendor/moodlehq/moodle-local_moodlecheck';
+        $plugin = new MoodlePlugin($pluginlocation);
+        $directory = $this->moodle->getComponentInstallDirectory($plugin->getComponent());
+        if (!is_dir($directory)) {
+            // Copy plugin into Moodle if it does not exist.
+            $filesystem = new Filesystem();
+            $filesystem->mirror($plugin->directory, $directory);
+        }
+
+        $process = $this->execute->passThroughProcess(
+            ProcessBuilder::create()
+                ->setPrefix('php')
+                ->add('local/moodlecheck/cli/moodlecheck.php')
```

Comment:
I'm guessing this script doesn't take a list of files?  There is a built in way to ignore files (if you look at other commands like PHPLint).  Would be nice if this, at the very least, skipped vendors defined in `thirdpartylibs.xml` or whatever it is called.

Reply:
Good point. it actually can take a list. I changed code to address this in 9dc2c9e